callouts/python/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM marketplace.gcr.io/google/debian11
+WORKDIR /home/callouts/python
+COPY . ./
+RUN apt-get update && apt-get install -y python3-grpc-tools
```

Comment:
I would probably move this line before WORKDIR - if we want to have more generic layers lower in the stack